js/components/DeploymentComponent.jsx

```diff
@@ -1,13 +1,15 @@
 /** @jsx React.DOM */
 
+var lazy = require("lazy.js");
```

Comment:
Ony for better understanding the rule is:

Start a variable with a capital letter if it is a class (with constructor) that is created with the ```new```-keyword.
Otherwise for functions/objects/etc. use lowercase letters.